src/css/vocabulary/vocabulary.css

```diff
@@ -161,7 +161,7 @@ header {
     box-sizing: border-box;
 
     text-decoration: none;
-    font-family: "Accidenz Commons";
+    font-family: "CC Accidenz Commons"; /* replaced with font-family: "CC Accidenz Commons"; */
```

Comment:
I don't think you need this comment, the record will be stored in the Git history.